apollo-federation/tests/query_plan/build_query_plan_tests.rs

```diff
@@ -541,75 +541,25 @@ fn key_where_at_external_is_not_at_top_level_of_selection_of_requires() {
           }
         "#,
         @r###"
-        QueryPlan {
-          Sequence {
-            Fetch(service: "A") {
-              {
-                u {
-                  __typename
-                  k1 {
-                    id
-                  }
-                }
+    QueryPlan {
```

Comment:
These tests are broken on `dev` and intentionally ignored with `should_panic`. Lenny reached out to the owner but I'm not sure if they have a plan to fix this. But for the `connectors` branch, we need to minimize changes to only things related to connectors, so you should probably remove this diff from your PR. I've just been running 'cargo insta reject` to clear out these snapshots, then running just the connectors tests that changed, then `cargo insta review`. That avoids picking up these snapshots.

Reply:
I do `cargo t validation -p apollo-federation` when working on validations to avoid even compiling router 😁

Reply:
Definitely didn't mean for this test to get included; I think I had happy fingers when going through skipping and approving 😅

ty for the catch!